lmeval-evaluation-job.adoc

```diff
@@ -0,0 +1,185 @@
+* `card`: Use the `name` to specify a Unitxt card or `custom` for a custom card
+** `name`: Specify a Unitxt card from the link:https://www.unitxt.ai/en/latest/catalog/catalog.__dir__.html[Unitxt catalog]. Use the card's ID as the value. For example: The ID of link:https://www.unitxt.ai/en/latest/catalog/catalog.cards.wnli.html[Wnli card] is `cards.wnli`.
+** `custom`: Define a custom card and use it. The value is a JSON string for a custom Unitxt card which contains the custom dataset. Use the link:https://www.unitxt.ai/en/latest/docs/adding_dataset.html#adding-to-the-catalog[Unitxt documentation] to compose a custom card, store it as a JSON file, and use the JSON content as the value here. If the dataset used by the custom card needs an API key from an environment variable or a persistent volume, you have to set up corresponding resources under the `pod` field. Check the `pod` field below.
+* `template`: Specify a Unitxt template from the link:https://www.unitxt.ai/en/latest/catalog/catalog.__dir__.html[Unitxt catalog]. Use the template's ID as the value.
+* `task` (optional): Specify a Unitxt task from the link:https://www.unitxt.ai/en/latest/catalog/catalog.__dir__.html[Unitxt catalog]. Use the task's ID as the value. A Unitxt card has a predefined task. Only specify a value for this if you want to run a different task.
+* `metrics` (optional):  Specify a Unitxt task from the link:https://www.unitxt.ai/en/latest/catalog/catalog.__dir__.html[Unitxt catalog]. Use the metric's ID as the value. A Unitxt task has a set of pre-defined metrics. Only specify a set of metrics if you need different metrics.
+* `format` (optional): Specify a Unitxt format from the link:https://www.unitxt.ai/en/latest/catalog/catalog.__dir__.html[Unitxt catalog]. Use the format's ID as the value.
+* `loaderLimit` (optional): Specifies the maximum number of instances per stream to be returned from the loader (used to reduce loading time in large datasets).
+* `numDemos` (optional): Number of fewshot to be used.
+* `demosPoolSize` (optional): Size of the fewshot pool.
+
+
+| `numFewShot`
+| Sets the number of few-shot examples to place in context. If you are using a task from Unitxt, don't use this field. Use `numDemos` under the `taskRecipes` instead.
+
+| `limit`
+| Instead of running the whole dataset, set a limit to run the tasks. Accepts an integer, or a float between 0.0 and 1.0.
+
+| `genArgs`
+| Map to `--gen_kwargs` parameter for the lm-evaluation-harness. Read more in the link:https://github.com/EleutherAI/lm-evaluation-harness/blob/main/docs/interface.md#command-line-interface[lm-evaluation-harness documentation].
+
+| `logSamples`
+| If this flag is passed, then the model's outputs, and the text fed into the model, will be saved at per-document granularity.
+
+| `batchSize`
+| Batch size for the evaluation. The auto:`N` batch size is not used for API models, but numeric batch sizes are used for APIs. Only int batch size supported at the moment.
```

Comment:
```suggestion
| Specifies the batch size for the evaluation in integer format. The `auto:N` batch size is not used for API models, but numeric batch sizes are used for APIs. 
```
Should this be formatted this way? Also guessing at what "int" means here but feel free to correct if I'm off